controller/handler/trial.js

```diff
@@ -59,14 +69,16 @@ function trialView (request, reply) {
                 JOIN stage AS st
                 ON st.id = pa.stageId
                 WHERE st.trialId = ?
-                AND si.endTime > ?
+                AND si.startTime > ?
+                AND si.endTime < ?
                 GROUP BY pa.id
                 `,
                 {
                     type: database.sequelize.QueryTypes.SELECT,
                     replacements: [
                         request.params.id,
-                        startDate.toISOString()
+                        fromDatestr,
+                        toDatestr
```

Comment:
I first did it in the database replacement but it gave error saying "Cannot convert toISOString of undefined"